js/main.js

```diff
@@ -1,2 +1,91 @@
 $(function () {
+  let random;
+
+  let stringBotScore = $("#computerScore").text()
+  let computerScore = parseInt(stringBotScore);
+  computerScore = 0;
+
+  let stringPlayerScore = $("#humanScore").text()
+  let humanScore = parseInt(stringPlayerScore);
+  humanScore = 0;
+
+
+  $("#rock").click(() => {
```

Comment:
@cbrooks2019 be careful with your indentation here (also with the other event listeners below)